libs/MobileSync/src/com/salesforce/androidsdk/mobilesync/app/MobileSyncSDKManager.java

```diff
@@ -83,7 +85,10 @@ private static void init(Context context, Class<? extends Activity> mainActivity
      * @param context      Application context.
      * @param mainActivity Activity that should be launched after the login flow.
      */
-    public static void initNative(Context context, Class<? extends Activity> mainActivity) {
+    public static void initNative(
+            @NonNull Context context,
```

Comment:
Be aware the Android Studio inspector does post warnings for missing nullability annotations downstream of the new Kotlin code.  I wouldn't call that an actual API change, though.  If anyone feels differently, let me know.

Reply:
I only corrected nullability warnings in sources that were already modified for another purpose in this pull request.  My hope was that would keep the difference report readable.  We can correct additional sources as they're modified in the future.

Reply:
You wouldn't put deprecations for a`@NonNull` annotation anyway so I think it's fine.  A major release, like 12.0, is the best time for these exact kind of changes.